Kucoin.Net/Objects/Models/Spot/KucoinOrder.cs

```diff
@@ -44,12 +44,12 @@ public class KucoinOrder: KucoinOrderBase
         /// The stop condition
         /// </summary>
         [JsonConverter(typeof(StopConditionConverter))]
-        public StopCondition Stop { get; set; }        
+        public StopCondition? Stop { get; set; }
```

Comment:
Made nullable for consistency, although revised StopConditionConverter never returns null